node_test/helpers_test.go

```diff
@@ -250,19 +250,36 @@ func checkPaymentChannel(t *testing.T, id types.Destination, o outcome.Exit, sta
 }
 
 // createLedgerInfo constructs a LedgerChannelInfo so we can easily compare it to the result of GetLedgerChannel
-func createLedgerInfo(id types.Destination, outcome outcome.Exit, status query.ChannelStatus) query.LedgerChannelInfo {
-	clientAdd, _ := outcome[0].Allocations[0].Destination.ToAddress()
-	hubAdd, _ := outcome[0].Allocations[1].Destination.ToAddress()
+func createLedgerInfo(id types.Destination, outcome outcome.Exit, status query.ChannelStatus, user types.Address) query.LedgerChannelInfo {
+	firstParticipant, _ := outcome[0].Allocations[0].Destination.ToAddress()
+	secondParticipant, _ := outcome[0].Allocations[1].Destination.ToAddress()
```

Comment:
Out of scope: are we swallowing errors here? Chore, but we might want to just panic (given it is test code) if there are errors.

Reply:
c757809e50d405dc50a87fe34484747a8ab90abb